src/modules/translation/languages/en.json

```diff
@@ -32,8 +32,9 @@
       "title": "Basic Controls",
       "undo": "Undo",
       "redo": "Redo",
-      "duplicate": "Duplicate object",
-      "delete": "Delete object",
+      "reset": "Reset item",
```

Comment:
May be a better name? @nicosantangelo 
`Reset item transform`
`Reset item rotation`